processing/src/main/java/org/apache/druid/frame/read/columnar/ColumnPlus.java

```diff
@@ -19,6 +19,7 @@
 
 package org.apache.druid.frame.read.columnar;
 
+import com.google.common.base.Supplier;
```

Comment:
unfortunately it needs to be the old one because that is what all of the `SimpleColumnHolder`/`ColumnBuilder` types are, unless you're suggesting we change those. I can add a default implementation though of this::getColumn.

I do want to change this in the future, in my experiments in #16132 I introduced a `ColumnSupplier` dedicated type instead of using a plain `Supplier`, i'll hopefully be revisiting it soon since it would be helpful for providing insight into the size information on projections.

Reply:
No, I'm not suggesting we change the SimpleColumnHolder/ColumnBuilder types. You can ignore this nit, it's just something that came to mind when seeing the guava version of the supplier.